app/gui2/stories/GraphEditor.story.vue

```diff
@@ -32,6 +35,20 @@ main =
     result = run_main benches
     third_node = 2 + 2
 `)
+
+/// Note: These props should be synced with the props in
```

Comment:
jsdocs are formatted like this:
```ts
/**
 * docs here
 */
```
otherwise just a double slash on the first line is fine